ros_ign_bridge/src/bridge_types.cpp

```diff
@@ -0,0 +1,113 @@
+// Copyright 2022 Open Source Robotics Foundation, Inc.
```

Comment:
when I run 
```bash
ros2 run ros_ign_bridge bridge_types --get-ign-to-ros ignition.msgs.UI
```

There is no output because the type doesn't exists, but I don't know if something happens in the background. Can you add a message? maybe something like "This type is not available use the option --print-ign to show the available messages"

Reply:
It returns non-zero, which is the most helpful for scripting, but I can add a human-readable message as well.